website/content/posts/introducing-the-enterprise-contract.md

```diff
@@ -0,0 +1,295 @@
+  - The cosign claims were validated
+  - The signatures were verified against the specified public key
+[…]
+```
+
+The SLSA Provenance contains a lot of useful information about the build process. The cosign
+verify-attestation command does offer some support for evaluating its contents with a policy engine.
+The current options are [Rego](https://www.openpolicyagent.org/docs/latest/policy-language/) and
+[Cue](https://cuelang.org/). Applying policies directly with cosign is possible, but let's introduce
+a tool to make it easier and more convenient.
+
+## Verifying with the Enterprise Contract
+
+The steps on the previous section work wonderfully when verifying an image. But what if you want to
+validate a group of images? And ensure they all meet the requirements of a certain policy? What if
+you want a configuration that can be shared across different teams?
+
+This is where the Enterprise Contract comes into play!
+
+The Enterprise Contract can be evaluated via the
+[ec-cli](https://github.com/enterprise-contract/ec-cli). The simplest example involves using an
+empty policy:
+
+```text
+$ ec validate image --policy '' --rekor-url '' --public-key cosign.pub --image $IMAGE --output yaml
+
+components:
+- containerImage: quay.io/lucarval/demo@sha256:304040ca1911aa4d911bd7c6d6d07193c57dc49dbc43e63828b42ab204fb1b25
+  name: Unnamed
+  signatures:
+  - keyid: SHA256:IhiN7gY+Z3uSSd7tmj6w5Zfhqafzdhm3DZjIvGc6iYY
+    metadata:
+      predicateBuildType: tekton.dev/v1beta1/TaskRun
+      predicateType: https://slsa.dev/provenance/v0.2
+      type: https://in-toto.io/Statement/v0.1
+    sig: MEUCIQDcgZIwEkLFqD7U9HrobgEC8Jo7wm+xJ5AoyO3qg+aj8QIgb9xDpjYGRMmpVk+QATeVKlHonzBiu51HtT3J+lQXPXc=
+  - keyid: SHA256:IhiN7gY+Z3uSSd7tmj6w5Zfhqafzdhm3DZjIvGc6iYY
+    metadata:
+      predicateBuildType: tekton.dev/v1beta1/PipelineRun
+      predicateType: https://slsa.dev/provenance/v0.2
+      type: https://in-toto.io/Statement/v0.1
+    sig: MEYCIQDKSihaAR/zAhJhR5GCqleDvfUUtvRw61vk0YeTBAnOSQIhAKa09B4yEfaSJronmWBFbu5cVPNxm17CMl/PElEz1POa
+  success: true
+ec-version: v0.1.1294-84267e8
+key: |
+  -----BEGIN PUBLIC KEY-----
+  MFkwEwYHKoZIzj0CAQYIKoZIzj0DAQcDQgAEZP/0htjhVt2y0ohjgtIIgICOtQtA
+  naYJRuLprwIv6FDhZ5yFjYUEtsmoNcW7rx2KM6FOXGsCX3BNc7qhHELT+g==
+  -----END PUBLIC KEY-----
+policy:
+  publicKey: cosign.pub
+success: true
+```
+
+The ec-cli has different output formats. I chose the one that displays the full report in YAML
+format. From its success, we can tell that the image signature and the image attestations match the
+provided public key. We can also see some metadata information about the found attestations.
+
+Let’s take a step further and say we want to validate the SLSA Provenance attestations with a
+certain set of policy rules. The [ec-policies](https://github.com/enterprise-contract/ec-policies)
+repo has a set of useful rego policies, so let’s use that.
+
+Create a policy file called `policy.yaml` with the following content:
+
+```text
```

Comment:
If we do this, does it highlight?

```suggestion
```yaml
```

Reply:
I think we should it regardless. Maybe style it better later if we want. I also broke down the last call to the ec-cli into two blocks, one with the command (text) and one with the output (yaml). That looks a bit better.